python/generational/decisions.py

```diff
@@ -72,7 +75,7 @@ def __init__(self, id):
         self.mother = None
         self.partner = None
         self.children = 0
-        self.sex = id % 2   # 0 for female, 1 for male
+        self.sex = _id % 2   # 0 for female, 1 for male
```

Comment:
id is a build in keyword, I'm changing this into a internal variable

Reply:
I'm going to change this to 'identity' to be more thorough